zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -617,39 +624,39 @@ function PageWrapper(props) {
                     {props.auth.tags.filter(
                       tag => tag === 'staff' || tag === 'educator',
                     ).length > 0 && (
-                        <MenuItem
-                          className={clsx(common_classes.removeOnSmallScreen)}
-                          onClick={() => {
-                            history.push('/activities', { flag: 'educator' });
-                          }}
+                      <MenuItem
```

Comment:
For URLs Its better to use anchor tags instead of using MenuItem for accessibility reasons and you will miss some options like open in a new tab etc.